src/autoschedulers/adams2019/LoopNest.h

```diff
@@ -34,6 +32,10 @@ bool may_subtile();
 std::vector<std::vector<int64_t>> generate_tilings(const vector<int64_t> &s, int d, int factor, bool allow_splits);
 
 struct LoopNest {
+    explicit LoopNest(bool may_subtile)
```

Comment:
may_subtile is essentially a flag for a comparison to prior work we wanted to do in the paper. It's pretty tedious to have to plumb it everywhere and wasteful to allocate space in the LoopNest struct to store it. I see that the params struct is already plumbed through into the child-generation routine, so why is this even necessary?

Reply:
Let me recheck -- when I did this I don't think that struct was plumbed everywhere it needed to go.